tests/jni_api.rs

```diff
@@ -534,6 +534,24 @@ pub fn get_primitive_array_critical() {
     assert_eq!(res[2], 4);
 }
 
+#[test]
+pub fn get_primitive_array_critical_commit() {
+    let env = attach_current_thread();
+
+    // Create original Java array
+    let java_array = env
+        .new_long_array(3)
+        .expect("JNIEnv#new_long_array must create a java array with given size");
+
+    // Get primitive array elements auto wrapper
+    let mut auto_ptr = env
+        .get_primitive_array_critical(java_array, ReleaseMode::CopyBack)
+        .unwrap();
+
+    // Call commit
+    auto_ptr.commit().unwrap();
```

Comment:
Shan't we have any assertions here (e.g., modify the first element, check that it's there)? However, the JNI spec does not allow calling other JNI methods whilst holding a critical array, so it might not be possible 🤔

Reply:
Exactly. That's why I wanted to do a minimal test. Just to confirm that the bug is there. And, that it's not related to calling any other JNI methods (at least, explicitly).